server.js

```diff
@@ -1,12 +1,57 @@
 const express = require('express');
 const app = express();
+const morgan = require("morgan");
 
+// Middlewares
+app.use(express.json());
+app.use(morgan('dev'));
+
+// Custom Middlewares
+const setPostMiddleware = (req,res,next)=>{
+  console.log(req.body)
+  next();
+}
+
+const getFullName = (req,res,next)=>{
+
```

Comment:
Nicely handled all possible error scenarios. This is actually already a mini validation middleware. 
So you are well prepared for the validation tasks we do in 2 weeks ;)